build-tests/install-test-workspace/workspace/common/pnpm-lock.yaml

```diff
@@ -5,27 +5,27 @@ importers:
   typescript-newest-test:
```

Comment:
This shouldn't be in the PR either.

Reply:
Will remove these and the test removals. Not sure why they even showed up on the build.

Reply:
This is updated from `rush update`